test/ci/buildkite/ngtf-cpu_ubuntu-external-fork.yaml

```diff
@@ -79,7 +79,7 @@
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
       pip install pytest
-      PYTHONPATH=`pwd`:`pwd`/tools:`pwd`/examples/mnist python3 test/ci/buildkite/test_runner.py \
+      PYTHONPATH=`pwd`:`pwd`/examples:`pwd`/tools:`pwd`/examples/mnist python3 test/ci/buildkite/test_runner.py \
```

Comment:
we can remove ``pwd`/examples/mnist` since we are adding `pwd`/examples

Reply:
@sayantan-nervana I think pwd/examples/mnist might still be needed, what do you think?

Reply:
I think for PYTHONPATH all paths need to be added separately, so we need to keep both

Reply:
ok.